contentcuration/contentcuration/frontend/shared/vuex/indexedDBPlugin/index.js

```diff
@@ -110,16 +109,15 @@ export default function IndexedDBPlugin(db, listeners = []) {
 
   db.on('changes', function(changes) {
     changes.forEach(function(change) {
-      // Don't invoke listeners if their client originated the change
-      if (CLIENTID !== change.source) {
```

Comment:
I think this might make the `CLIENTID` as a transaction source superfluous, but I'd have to double check.